src/symmbasis.jl

```diff
@@ -107,10 +104,7 @@ function _get_ordered(bb, invAspec)
 end
 
 
-function coupling_coeffs(bb, rotc::Rot3DCoeffs, φ::Invariant)
-   # bb = [ b1, b2, b3, ...)
-   # bi = (μ = ..., n = ..., l = ..., m = ...)
-   #    (μ, n) -> n; only the l and m are used in the angular basis
+function coupling_coeffs(bb, rotc::Rot3DCoeffsEquiv, φ::Invariant)
```

Comment:
same here, I would prefer to keep these comments

Reply:
ok, I think I may have deleted these by accident...